docs/source/concept_models/vivarium_swissre_stomachcancer/concept_model.rst

```diff
@@ -0,0 +1,440 @@
+ -  Cardia cancers are not associated with H. pylori infection while non-cardia cancers are. There is evidence written that says cardia cancers may be protected by H. pylori. Incidence is cardia cancers   are rising while stomach cancer overall and non-cardia cancers are decreasing (better hygiene?) 
+ - This pools all stomach cancer
+
+
+.. image:: correa_cascade.svg
+
+.. image:: stomachcancer_lifehistory.svg
+
+.. _1.1:
+
+1.1 Project overview
+--------------------
+
+
+.. _1.2:
+
+1.2 Literature review
+---------------------
+
+
+
+.. _2.0:
+
+2.0 Modeling aims and objectives
+++++++++++++++++++++++++++++++++
+
+To estimate the yearly number of cases of stomach cancer detected under specific screening practices and the yearly number of deaths from undetected stomach cancer (both in unit of per 100,000 insured person-years) in order to identify pay-out trends for critical insurance claims (CII).  
+
+.. _3.0:
+
+3.0 Causal framework
+++++++++++++++++++++
+
+.. _3.1:
+
+3.1 Causal diagram
+------------------
+
+**Outcome (O)**:
+
+  - stomach cancer 
+
+**Exposure (E)**:
+  
+  - h.pylori
+
+
+
+.. _3.2:
+
+3.2 Effect and impact size
+--------------------------
+
+RR of H. pylori = 4.5 
+Exposure of H. pylori = 44%
+PAF of H. pylori = 60%
+
+.. _4.0:
+
+4.0 Intervention
+++++++++++++++++
+
+Scale-up of stomach cancer screening coverage among insured population 
+
+.. _4.1:
+
+4.1 Simulation scenarios
+------------------------
+
+:underline:`Baseline scenario`
+
+  * no screening coverage
+
+:underline:`Alternative scenario`
+
+30% of insured Chinese male/female initiated stomach cancer screening in 2020, stay 30% for one year then linearly project to 75% by 2030 and hold constant till 2040 for blended provinces, where:
+
+  * same screening mechanisms as compared to baseline for different age groups and risk exposure level.
+
+ 
+
+.. _5.0:
+
+5.0 Vivarium modelling strategy
++++++++++++++++++++++++++++++++
+
+.. _5.1:
+
+5.1 Vivarium concept model 
+--------------------------
+
+.. image:: vivarium_concept_model_diagram_stomachcancer.svg
+
+.. _5.2:
+
+5.2 Demographics
+----------------
+
+.. _5.2.1:
+
+5.2.1 Population description
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* Cohort type: Closed cohort of 200,000 insured male (50%) and female (50%) simulants
+* Age and sex: Aged 15 to 95+, 5 year-age bands, uniformly distributed age and sex structure
+* Time span: Jan 1, 2020 to Dec 31, 2040 with 36.5-day time-steps. 
+* Currently assume the sim population buys insurance on the first day of sim start. This means no one has prior insurance and were paid out for their cancers before sim start. 
+
+.. _5.2.2:
+
+5.2.2 Location description
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+*Potential* provinces to model include Tianjin, Jiangsu, Guangdong, Henan, and Heilongjiang (optional). The same population distribution of age and sex will be used among the different provinces.
+
```

Comment:
We should remove "potential" at this point since we decided to include all provinces listed in the province blend. This is an edit that has not been made yet in the breast cancer documentation.